idea-plugin/src/main/kotlin/arrow/meta/ide/testing/IdeTest.kt

```diff
@@ -5,6 +5,12 @@ import com.intellij.testFramework.fixtures.CodeInsightTestFixture
 
 typealias Source = String
 
+/**
+ * [IdeTest] is a polymorphic aggregation of one complete test suite.
+ * [myFixture] is a key component of the underlying IntelliJ Testing environment.
+ * [test] defines what exact test is run on the [code]
+ * [result] describes the expected shape of [A] with a costume message.
```

Comment:
```suggestion
 * [result] describes the expected shape of [A] with a custom message.
```
"costume" or "custom"?

Reply:
true haha. Thank you @JorgeCastilloPrz